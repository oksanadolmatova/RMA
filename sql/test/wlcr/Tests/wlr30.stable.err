stderr of test 'wlr30` in directory 'sql/test/wlcr` itself:


# 10:21:36 >  
# 10:21:36 >  "/usr/bin/python2" "wlr30.py" "wlr30"
# 10:21:36 >  

# builtin opt 	gdk_dbpath = /export/scratch1/mk/wlcr//Linux/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	monet_prompt = 
# cmdline opt 	mapi_port = 55121
# cmdline opt 	mapi_usock = /var/tmp/mtest-29054/.s.monetdb.55121
# cmdline opt 	gdk_dbpath = /export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr-clone
# cmdline opt 	gdk_debug = 536870922

# 10:21:36 >  
# 10:21:36 >  "Done."
# 10:21:36 >  

