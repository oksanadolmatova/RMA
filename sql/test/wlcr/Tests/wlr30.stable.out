stdout of test 'wlr30` in directory 'sql/test/wlcr` itself:


# 10:21:36 >  
# 10:21:36 >  "/usr/bin/python2" "wlr30.py" "wlr30"
# 10:21:36 >  

# MonetDB 5 server v11.26.0 (hg id: cbfc9f47e58c)
# This is an unreleased version
# Serving database 'mTests_sql_test_wlcr-clone', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 15.589 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://vienna.da.cwi.nl:55121/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-29054/.s.monetdb.55121
# MonetDB/GIS module loaded
#replay log file:/export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr-clone/../mTests_sql_test_wlcr/master/mTests_sql_test_wlcr_000000000010
#replay log file:/export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr-clone/../mTests_sql_test_wlcr/master/mTests_sql_test_wlcr_000000000011
#replay log file:/export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr-clone/../mTests_sql_test_wlcr/master/mTests_sql_test_wlcr_000000000012
#replay log file:/export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr-clone/../mTests_sql_test_wlcr/master/mTests_sql_test_wlcr_000000000013
# MonetDB/SQL module loaded
#select * from tmp;
% sys.tmp,	sys.tmp # table_name
% i,	s # name
% int,	clob # type
% 1,	5 # length
[ 1,	"hello"	]
[ 2,	"world"	]
[ 3,	"blah"	]
[ 4,	"bloh"	]
[ 5,	"red"	]
[ 6,	"fox"	]
#call replicate('mTests_sql_test_wlcr');
#Waiting for replay scheduler to stop
#replay log file:/export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs//mTests_sql_test_wlcr_000000000004
#replay log file:/export/scratch1/mk/wlcr//Linux/var/MonetDB/mTests_sql_test_wlcr/wlc_logs//mTests_sql_test_wlcr_000000000005
#select * from tmp;
% sys.tmp,	sys.tmp # table_name
% i,	s # name
% int,	clob # type
% 1,	4 # length
[ 3,	"blah"	]
[ 2,	"blah"	]
[ 3,	"blah"	]
[ 4,	"blah"	]
[ 5,	"blah"	]
[ 6,	"blah"	]

# 10:21:36 >  
# 10:21:36 >  "Done."
# 10:21:36 >  

